### ISSUE TITLE
@unchecked Sendable wrap

### ISSUE THREAD

**Issue by kokluch** [nicklockwood/SwiftFormat#1341]
Feel off that @unchecked Sendable is wrap when using --typeattributes prev-line

**Comment by nicklockwood** [comment]
@kokluch agreed, thanks for reporting

**Comment by kokluch** [comment]
Also find this replacement that doesn't feel right.
-    let closure: @MainActor () -> Void
+    let handler: @MainActor ()
+        -> Void

**Comment by OlegEremenko991** [comment]
> Also find this replacement that doesn't feel right.
> 
> ```swift
> -    let closure: @MainActor () -> Void
> +    let closure: @MainActor ()
> +        -> Void
> ```

I have the same issue

**Comment by sigsbeym** [comment]
+1

```
    private var closure: @MainActor (SomeResponseType) -> Void
    private var closure: @MainActor
    (SomeResponseType) -> Void
```

**Comment by nicklockwood** [comment]
@kokluch @sigsbeym what config are you using to trigger this wrap? I'm not seeing it with just `--typeattributes prev-line`

**Comment by kokluch** [comment]
@nicklockwood hi, here is the exhaustive list of arguments I'm using.

```
            --swiftversion 5
            --ifdef outdent
            --wraparguments before-first
            --wrapparameters before-first
            --wrapcollections before-first
            --funcattributes prev-line
            --typeattributes prev-line
            --commas inline
            --enable isEmpty
            --enable blankLinesBetweenScopes
            --enable blankLinesBetweenImports
            --enable blankLineAfterImports
            --enable preferDouble
            --enable blockComments
            --enable wrapEnumCases
            --enable wrapSingleLineComments
            --disable wrapMultilineStatementBraces
            --header strip
            --self init-only
            --decimalgrouping 3
            --stripunusedargs closure-only
            --extensionacl on-declarations
            --importgrouping testable-last
```

**Comment by sigsbeym** [comment]
--disable extensionAccessControl
--disable fileHeader

**Comment by sigsbeym** [comment]
```
--disable extensionAccessControl
--disable fileHeader
--disable initCoderUnavailable
--disable modifierOrder
--disable numberFormatting
--disable preferKeyPath
--disable sortDeclarations
--disable trailingClosures
--disable unusedArguments
--disable wrap
--enable blockComments
--enable isEmpty
--patternlet inline
--funcattributes prev-line
--typeattributes prev-line
--varattributes preserve
--ranges no-space
--wraparguments before-first
--wrapparameters before-first
--wrapcollections before-first
--wrapconditions before-first
--wraptypealiases before-first
--indent 4
--tabwidth 4
--smarttabs disabled
--xcodeindentation true
--guardelse next-line
```

**Comment by sigsbeym** [comment]
> @kokluch @sigsbeym what config are you using to trigger this wrap? I'm not seeing it with just `--typeattributes prev-line`

Pretty sure it's `--funcattributes prev-line`

**Comment by nicklockwood** [comment]
OK, I managed to repro. It only happens when there are multiple @MainActor attributes or properties is sequence.

**Comment by nicklockwood** [comment]
@kokluch @OlegEremenko991 @sigsbeym fixed in 0.50.9
